cdist/conf/type/__rsync/man.text

```diff
@@ -73,12 +80,18 @@ __rsync otherstuff \
     --destination /usr/local/bin \
     --source "$__type/files/package2"
 
+# Use rsync option --exclude
+__rsync /tmp/testdir \
+    --source /etc \
+    --rsync-opts exclude=sshd_conf
+
```

Comment:
Maybe add a second example with additional --rsync-opts to make it clear that more than one is supported